medical-practitioner-portal/src/app/case-details/case-details.component.ts

```diff
@@ -119,6 +128,36 @@ export class CaseDetailsComponent implements OnInit {
         if (caseDetails.status === 'Closed') {
           this.selectedIndex = 5;
         }
+
+        // Load docuemnts
+        this.getdriverdocuments(this.caseDetails?.driverId as string);
+      });
+  }
+
+  getdriverdocuments(driverId: string) {
```

Comment:
use camelCasing and you can delete the console.log line

Reply:
Fixed the method name to camelCase